keyboards/handwired/polykybd/lang/_gen_compat62_cols.py

```
@@ -0,0 +1,106 @@
+for name, src in SPECMAP:
+    if src == FOLD:
+        SPEC[name] = dict(SET_FOLD)
+    else:
+        assert src in langs, f"source column {src} not found"
+        SPEC[name] = clone_col(sh, langs, src)
+
+ORDER = [n for n, _ in SPECMAP]
+assert len(ORDER) == len(set(ORDER)) == 62, f"expected 62 unique, got {len(ORDER)}"
+
+spec_json = {n: {str(r): v for r, v in cols.items()} for n, cols in SPEC.items()}
+json.dump(spec_json, open("/tmp/compat62_cols.json", "w"), indent=1)
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/compat62_cols.json"

(S108)